composer.json

```diff
@@ -14,7 +14,7 @@
         }
     ],
     "require": {
-        "php": "^5.3.6 || >=7.0 <7.2",
+        "php": "^5.3.6 || >=7.0 <7.3",
```

Comment:
just because of this bump, do we know that integration test of 7.2 (https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/tests/Fixtures/Integration/misc/PHP7_2.test) contains all syntax changes added in 7.2 ?

Reply:
I couldn't find any more new syntax changes, but a second look wouldn't hurt.
There are more opportunities for fixing towards PHP7.2, but I couldn't find more breaking issues.

Reply:
ref https://github.com/SpacePossum/PHP-CS-Fixer/pull/20